src/ncp/ncp_base.cpp

```diff
@@ -2831,7 +2831,7 @@ otError NcpBase::GetPropertyHandler_THREAD_ON_MESH_NETS(uint8_t header, spinel_p
             64,
             border_router_config.mStable,
             flags,
-            true
+            false
```

Comment:
We first get the prefixes from non-local network data (passing `false` for `aLocal` input  in otNetDataGetNextPrefixInfo(mInstance, false, &iter, &border_router_config);`....